android/build.gradle

```diff
@@ -1,3 +1,31 @@
+import groovy.json.JsonSlurper
+
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
+def warnNoAutoInit() {
+    logger.warn("Couldn't auto-initialize the Heap library; manually initialize it by following the instructions at https://docs.heap.io/docs/react-native.")
+}
+
+def warnDisabled() {
+    logger.warn("Not auto-initializing the Heap library because it was manually disabled; set `enabled` to `true` in `heap.json` to enable auto-initialization.")
+}
+
+def heapAppId() {
+    def heapConfigFilename = 'heap.json'
```

Comment:
Could this be a CONSTANT at the top of the file?

Reply:
I attempted doing this for about 5 minutes but it didn't work for some reason. I'm not sure if Gradle+Groovy has constants per se, but I could just move this `def` up to the global scope I suppose.